src/auth/auth.controller.spec.ts

```diff
@@ -77,6 +108,35 @@ describe("AuthController", () => {
           });
         });
     });
+
+    it("should return a token and 201 http code when account information is correct.", async () => {
+      const expectedResult = {
+        accessToken: "mocked_token",
```

Comment:
may dry out this as a variable with L55

Reply:
Did you mean `jest.spyOn(localStrategy, "validate")`
Make this return mock result?

But this way will make can't return true or throw error.

Reply:
no, i mean...

```
const fakeAccessToken = "mocked_token";
...
  useValue: {
    // 模拟JwtService中的方法
    sign: jest.fn().mockReturnValue(fakeAccessToken),
  },
...
  accessToken: fakeAccessToken,
```

Reply:
Revised.
[refactor: make fake access token as a variable](https://github.com/Cophr/backend/pull/10/commits/658517fa64f21ff7b2b60b9939ed4f7479fd92ac)